frame/support/src/storage/bounded_vec.rs

```diff
@@ -191,54 +205,76 @@ impl<T, S: Get<u32>> BoundedVec<T, S> {
 		S::get() as usize
 	}
 
-	/// Forces the insertion of `s` into `self` retaining all items with index at least `index`.
+	/// Returns true of this collection is full.
+	pub fn is_full(&self) -> bool {
+		self.len() == Self::bound()
```

Comment:
```suggestion
		self.len() >= Self::bound()
```

We should be defensive :P

Reply:
`is_full_or_more_than_full`

Reply:
I think the name is okay ;)